website/content/en/guides/level-up/vector-tap-guide.md

```diff
@@ -0,0 +1,243 @@
+author_github: https://github.com/001wwang
+domain: operations
+weight: 6
+tags: ["cli", "tap", "level up", "guides", "guide"]
+---
+
+{{< requirement >}}
+Before you begin, this guide assumes the following:
+
+* You understand the [basic Vector concepts][concepts]
+* You understand [how to set up a basic pipeline][pipeline]
+
+[concepts]: /docs/about/concepts
+[pipeline]: /docs/setup/quickstart
+{{< /requirement >}}
+
+Vector's `tap` CLI command allows you to observe events as they flow to and from
+components in your pipelines. If you've ever attached a `console` sink to an
+output for debug purposes and wondered if there's a better way, you're in the
+right place. This guide walks you through how `vector tap` can be used to level
+up your troubleshooting experience.
+
+## Getting Started
+
+To start, we'll reference the following base configuration, but feel free to
+substitute your own. Just note that the [Vector API] must be enabled for `vector
+tap` to work. See [under the hood](#under-the-hood) for more details.
+
+```toml
+[api]
+enabled = true
+
+[sources.in]
+type = "demo_logs"
+format = "shuffle"
+lines = [
+  "test1",
+  "test2",
+]
+
+[sinks.out]
+type = "blackhole"
+inputs = ["in*"]
+```
+
+Run Vector with this configuration and watch the configuration for changes.
+
+```console
+vector --config path/to/config.toml -w
+```
+
+Now run `vector tap`! You should see start seeing a stream of
+notifications (sent to `stderr`) and events (sent to `stdout`) in your terminal.
+
+```console
+[tap] Pattern "*" successfully matched.
+{"message":"test1","source_type":"demo_logs","timestamp":"2022-02-22T19:20:40.487671258Z"}
+{"message":"test2","source_type":"demo_logs","timestamp":"2022-02-22T19:20:41.486858019Z"}
+...
+```
+
+Notifications are informative messages prefixed with `[tap]` and, in this case,
+events are logs flowing out of the `demo_logs` component `in`. There's a lot
+going on here -- component patterns, events from outputs, JSON formatting -- so
+let's start by unpacking it all.
+
+## Usage Basics
+
+### Component Patterns
+
+You can `tap` both the output events of components (sources or transforms) and
+the input events of components (transforms or sinks) by providing component ID
+patterns in the `--outputs-of` and `--inputs-of` options respectively. More
+specifically, glob patterns are accepted in these options. You can also specify
+output patterns as additional arguments.
+
+Running the bare `vector tap` command invokes sensible defaults and is
+equivalent to specifying `--outputs-of "*"` where `"*"` matches any component
+ID.
+
+Try running `vector tap --inputs-of "out"`. The events should be the same
+as before, but you're now tapping the input events of the `blackhole` sink
+`out`. Note that tapping a component's input is effectively a shorthand for
+tapping all the outputs that feed into that component.
```

Comment:
I feel like this note should get different formatting... 🤔

Reply:
Sure thing, I agree it's worth calling out more obviously.